doc/user/scripting.rst

```diff
@@ -26,3 +36,50 @@ In order to use scripting, FRR must be built with ``--enable-scripting``.
    contents of a script that is in use without restarting FRR. Not all
    scripting locations may behave this way; refer to the documentation for the
    particular location.
+
+
+Example: on_rib_process_dplane_results
+--------------------------------------
+
+This example shows how to write a Lua script that logs changes when a route is added.
+
+First, identify the Lua hook call to attach a Lua function to: this will be the name
+of the Lua function. In this case, since the hook call is
+`on_rib_process_dplane_results`:
+
+.. code-block:: lua
+
+   function on_rib_process_dplane_results(ctx)
+      log.info(ctx.rinfo.zd_dest.network)
+      return {}
+
+
+The documentation for :ref:`on-rib-process-dplane-results` tells us its arguments.
+Here, the destination prefix for a route is being logged out.
+
+Scripts live in :file:`/etc/frr/scripts/` by default. This is
+configurable at compile time via ``--with-scriptdir``. It may be
+overriden at runtime with the ``--scriptdir`` daemon option.
```

Comment:
IIRC you use vim - ^v, select the paragraph, `gqq` will reflow a paragraph for an 80 column line limit. I get 
```rst
Scripts live in :file:`/etc/frr/scripts/` by default. This is configurable at
compile time via ``--with-scriptdir``. It may be overriden at runtime with the
``--scriptdir`` daemon option.
```
Not a blocking comment though, just a nit.

Reply:
Ah. I reflowed the file